olp-cpp-sdk-core/src/http/android/NetworkAndroid.cpp

```diff
@@ -35,7 +35,10 @@
 
 namespace {
 constexpr auto kLogTag = "NetworkAndroid";
-}
+constexpr auto kUserAgentHeader = "User-Agent";
```

Comment:
We need to define this headers in some common place like NetworkUtils.h or NetworkConstants.h, e.g. https://github.com/microsoft/cpprestsdk/blob/master/Release/include/cpprest/details/http_constants.dat

Reply:
Done - see https://github.com/heremaps/here-olp-edge-sdk-cpp/pull/102